tests/memory_checker/test_memory_checker.py

```diff
@@ -542,7 +542,7 @@ def consumes_memory_and_checks_container_restart(duthost, container):
     loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix=marker_prefix)
     loganalyzer.expect_regex = container.get_restart_expected_logre()
     with loganalyzer:
-        timeout_monit_fail = 180  # fails happens after 10 cycles of 1 second
+        timeout_monit_fail = 360  # fails happens after timeout wait
```

Comment:
>360 [](http://example.com/codeflow?start=29&length=3)

Please ensure no impact to test run time for strong hardware SKU.

Reply:
<img width="751" alt="image" src="https://github.com/user-attachments/assets/864424b1-0e15-4ef8-8869-fca32b233fbd">
Verified some rounds on 7215, refer above we can see no runtime increase introduced.